sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_MockClient.md

```diff
@@ -0,0 +1,49 @@
+# Mock a client for testing using the Moq library
+
+This sample illustrates how to use [Moq][moq] to create a unit test that mocks the response from a `TextAnalyticsClient` method. For more examples of mocking, see [Moq samples][moq_samples].
+
+## Define method that uses TextAnalyticsClient
+To show the usage of mocks, define a method that will be tested with mocked objects. For this case, we are going to create a method that will verify if a document is writen in Spanish.
+
+```C# Snippet:MethodToTest
+private static async Task<bool> IsSpanishAsync(string document, TextAnalyticsClient client, CancellationToken cancellationToken)
+{
+    while (!cancellationToken.IsCancellationRequested)
```

Comment:
Why is this running in the loop? Would it be better to remove the loop and simply pass the token to the service method, i.e.
```csharp
DetectedLanguage language = await client.DetectLanguageAsync(document, cancellationToken);
```

Reply:
You are right! my bad.
Fix in https://github.com/Azure/azure-sdk-for-net/pull/10995